src/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "date-fns": "^2.16.1",
+    "firebase": "^8.0.2",
```

Comment:
Thanks Jasper. It might be best that you set up firebase since you have the most knowledge on it. I can probably figure it out but I fear I might introduce errors further down the line without realising/understanding what ive done.

Reply:
Okay, I can take a look at this now and get it going.

Reply:
Added my changes, closing this comment thread